gatsby-site/src/components/SunriseHeader.js

```diff
@@ -0,0 +1,42 @@
+import React from "react"
+import SunriseLogo from "./SunriseLogo"
+import styled from 'styled-components';
+
+import './global.css'
+
+const Container = styled.div`
+  display: flex;
+  flex-direction: row;
+  width: 100%;
+  background-color: var(--sunrise-grey);
+  height: inherit;
+  align-items: center;
+  padding-top: 25px;
+  padding-bottom: 25px;
+`;
+
+const Logo = styled.div`
+  padding-top: 3px;
+  height: 30px;
+  width: 18px;
+  margin-left: 8%;
```

Comment:
I only have one nitpicky thing:
I think this margin-left 8% should be moved to Container and be padding-left: 8% instead.

I was confused why we were pushing the Logo to the right until I deleted this line and realize we really just want the container's items to begin to the right!